site/viamguides/src/yolo-training/yolo-training.md

```diff
@@ -0,0 +1,307 @@
+author: HipsterBrown
+id: yolo-training
+summary: Train and deploy a custom YOLO object detection model
+categories: Getting-Started, Developer
+environments: web
+status: Published 
+feedback link: https://github.com/viam-devrel/viamcodelabs/issues
+tags: Getting Started, Developer
+
+# Train & run a custom YOLO model
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+In the world of computer vision, [YOLO](https://docs.ultralytics.com/#yolo-a-brief-history) (You Only Look Once) has revolutionized object detection with its speed and accuracy. Unlike traditional methods that scan images multiple times at different scales, YOLO divides images into a grid and predicts bounding boxes and class probabilities in a single pass – hence the name "You Only Look Once."
```

Comment:
This section contains a lot of text for the primary landing page, and I worry that it's not a digestible at first glance. 

- Would it be helpful to move some of the benefits to the last page, similar to how we have typically mentioned use cases and practical applications of what you're building on the last page?
- A picture is worth a thousand words. Consider moving the gif from What You'll Build above the fold. OR add a small technical diagram of training in the Viam cloud and model deploying to Macbook / device.